### PR TITLE
Dev/crash fix

### PR DESCRIPTION
**Pull request by Ishika1601-boxpay**

### CODE REVIEW

File: Sdk/src/main/java/com/boxpay/checkout/sdk/BoxPayCheckout.kt [hunk 1]
```diff
@@ -34,6 +35,24 @@ class BoxPayCheckout(
...
     )
 
+    constructor(
+        context: Context,
+        token: String,
+        onPaymentResult: ((PaymentResultObject) -> Unit)?,
+        customerShopperToken: String = "",
+        sandboxEnabled: Boolean = false,
+        isSuccessScreenVisible: Boolean = true,
+        extraParams: HashMap<String, Boolean>
```
Comment (aksain-boxpay) [inline Sdk/src/main/java/com/boxpay/checkout/sdk/BoxPayCheckout.kt]: should be configurationOptions

Comment (aksain-boxpay) [reply Sdk/src/main/java/com/boxpay/checkout/sdk/BoxPayCheckout.kt]: Lets make key as Enum - ConfigurationOptions with following value - 

1. SHOW_UPI_QR_ON_LOAD

**Review by aksain-boxpay — DISMISSED** [review]

File: Sdk/src/main/java/com/boxpay/checkout/sdk/BoxPayCheckout.kt (hunk 1 above)
Comment (aksain-boxpay) [reply Sdk/src/main/java/com/boxpay/checkout/sdk/BoxPayCheckout.kt]: Lets make it Map instead of HashMap

Comment (Ishika1601-boxpay) [reply Sdk/src/main/java/com/boxpay/checkout/sdk/BoxPayCheckout.kt]: done changes added

**Review by aksain-boxpay — DISMISSED** [review]

**Review by aksain-boxpay — APPROVED** [review]

**Pull request merged**